docs/tutorial/keystrokes.md

```diff
@@ -0,0 +1,78 @@
+---
+category: tutorial
+order: 80
+menu-title: Handling keystrokes
+---
+
+# Handling keystrokes
+
+## Accessibility
+
+Currently, our `highlight` plugin requires the user to click the button in the editor's toolbar to highlight selected text. This can be a problem for users who rely on the keyboard, so let's add a keyboard shortcut as an alternative way to highlight the text.
+
+## Adding keyboard shortcuts
+
+Common shortcut for highlighting text is <kbd>CTRL</kbd> + <kbd>ALT</kbd> + <kbd>H</kbd>, so that's what we're going to use in our plugin.
+
+To execute the `highlight` command when those keys are pressed, add the following code to the end of the `Highlight` function:
+
+```js
+editor.keystrokes.set( 'Ctrl+Alt+H', 'highlight' );
```

Comment:
Maybe we should mention that keystrokes on MacOS are automatically translated to `Cmd+Alt+H'?